### PR TITLE
ICONS: add more gob games to TOC.md

### PR DESCRIPTION
**Pull request by BJNFNE**

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -432,7 +432,7 @@
 | ✅ | gob | adi5                 | ADI 5                                                                       | GandalfTheWhite80 |
 | ✅ | gob | adibou1              | ADIBOU 1                                                                    | sdelamarre, GandalfTheWhite80 |
 | ✅ | gob | adibou2              | ADIBOU 2                                                                    | sdelamarre, GandalfTheWhite80 |
-| ✅ | gob | adibou3              | ADIBOU 3                                                                    | sdelamarre, GandalfTheWhite80, BJNFNE |
+| ✅ | gob | adibou3              | Adibou 3                                                                    | sdelamarre, GandalfTheWhite80, BJNFNE |
```
Comment (tag2015) [inline TOC.md:435]: Shouldn't you change Adibou 1/2 as well?

Comment (BJNFNE) [reply TOC.md:435]: ADIBOU 1 and ADIBOU 2 uses uppercase for some versions

Comment (BJNFNE) [reply TOC.md:435]: With Adibou 3, Coktel Vision began to write Adibou in lowercase.

Comment (tag2015) [reply TOC.md:435]: > ADIBOU 1 and ADIBOU 2 uses uppercase for some versions

All right, but like this it looks inconsistent. Also rename the other two or leave it as is. In any case this is merely a TOC and these minor changes aren't really needed

Comment (BJNFNE) [reply TOC.md:435]: then we may leave it as it is an revert /exclude the ADIBOU 3 related commit?

Comment (BJNFNE) [reply TOC.md]: revert commit is pushed

**Comment by tag2015** [comment]
Thanks!  Squashing

**Pull request merged**